scripts/run.sh

```diff
@@ -1,8 +1,9 @@
 set -euo pipefail
 script_dirpath="$(cd "$(dirname "${0}")" && pwd)"
 
-GECKO_IMAGE_DEFAULT="kurtosistech/gecko"
-CONTROLLER_IMAGE="kurtosistech/ava-e2e-tests_controller:latest"
+# This Docker image defines the gecko image used for developing e2e tests.
+GECKO_IMAGE_DEFAULT="avaplatform/gecko"
+CONTROLLER_IMAGE="avaplatform/avalanche-e2e-tests_controller:latest"
 root_dirpath="$(dirname "${script_dirpath}")"
 
 "${root_dirpath}/build/ava-e2e-tests" "--gecko-image-name=${GECKO_IMAGE_DEFAULT}" "--test-controller-image-name=${CONTROLLER_IMAGE}" ${*:-}
```

Comment:
Could you update this to avalanche-e2e-tests to reflect the name change to the binary?